ansible/roles/pyassimp/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+# Fix pyassimp issue
+
```

Comment:
please add here code which will make sure that /tmp/assimp folder doesn't exists. You can use this Ansible module http://docs.ansible.com/ansible/file_module.html